src/routes/activity.js

```diff
@@ -5,6 +5,8 @@ import validations from '../validations/activity';
 const activityRouter = Router();
 
 activityRouter
+  .get('/', activityController.getAllActivity)
+  .post('/', validations.validateUpdate, activityController.createActivity)
```

Comment:
do you have a function to validateCreate? the function Is almost the same, but the values should have require
```suggestion
  .post('/', validations.validateCreate, activityController.createActivity)
```

Reply:
Does not exist validateCreate. 
You would have to create the function and which fields are required

Reply:
I delete the validateCreate because it was the same of validateUpdate.

Reply:
validateCreate should have .required in every field, that is the difference

Reply:
when you create an activity, every field should be completed